api/controllers/node.js

```diff
@@ -86,35 +110,28 @@ NodeController.getConstants = function(context, next) {
  * @param {function} next
  * @todo Add description for the function and the params
  */
-NodeController.getStatus = function(context, next) {
-	return modules.node.shared.getStatus(null, (getStatusErr, data) => {
-		try {
-			if (getStatusErr) {
-				return next(getStatusErr);
-			}
-
-			data = _.cloneDeep(data);
-
-			// Check if properties are null, then set it to 0
-			// as per schema defined for these properties in swagger
-			data.networkHeight = data.networkHeight || 0;
-			data.consensus = data.consensus || 0;
-
-			return modules.transactions.shared.getTransactionsCount(
-				(getTransactionsCountErr, count) => {
-					if (getTransactionsCountErr) {
-						return next(getTransactionsCountErr);
-					}
-
-					data.transactions = count;
-
-					return next(null, data);
-				}
-			);
-		} catch (error) {
-			return next(error);
-		}
-	});
+NodeController.getStatus = async (context, next) => {
+	try {
+		const networkHeight = await getNetworkHeight({
+			normalized: false,
+		});
+
+		const data = {
+			broadhash: library.modules.system.getBroadhash(),
+			consensus: library.modules.peers.getLastConsensus() || 0,
+			currentTime: Date.now(),
+			secondsSinceEpoch: slots.getTime(),
+			height: library.modules.blocks.lastBlock.get().height,
```

Comment:
Either `library.modules.system.getHeight()` or `storage.blocks.getOne({...})` to remove dependency on `blocks`.

Reply:
Done